### ISSUE TITLE
npm-groovy-lint --version returns wrong CodeNarc version

### ISSUE THREAD

**Issue by djukxe** [nvuillam/npm-groovy-lint#334]
After upgrading to 13.0.0, `npm-groovy-lint --version` displays CodeNarc version 3.3.0

```shell
➜  ~ npm-groovy-lint --version
GroovyLint: Successfully processed CodeNarc:
CodeNarc version 3.3.0

npm-groovy-lint version 13.0.0

Embeds:
CodeNarc version 3.3.0
- Groovy version 3.0.9 (superlite)
```
While 13.0.0 seems to be based on newer version of CodeNarc which is a bit confusing
``
BREAKING CHANGE: Upgrade CodeNarc to v3.4.0-alpha+3346775f
``

**Comment by stevenh** [comment]
This is expected as that's how the CodeNarc build system works as the tag doesn't actually exist. This will be corrected when CodeNarc does an official release some time next year.
